OSDOCS#12138: Supported openshift versions for hosted clusters (#82541)
openshift/openshift-docs: modules/hcp-update-ocp-hc.adoc

// * hosted_control_planes/hcp-updating.adoc

:_mod-docs-content-type: PROCEDURE
[id="hcp-update-ocp-hc_{context}"]
= Updating a control plane in a hosted cluster

On {hcp}, you can upgrade your version of {product-title} by updating the hosted cluster. The `.spec.release` in the `HostedCluster` custom resource (CR) shows the version of the control plane. The `HostedCluster` updates the `.spec.release` field to the `HostedControlPlane.spec.release` and runs the appropriate Control Plane Operator version.

> The `HostedControlPlane` resource orchestrates the rollout of the new version of the Control Plane components along with the {product-title} component in the data plane through the new version of the cluster version operator (CVO). The `HostedControlPlane` includes the following artifacts:

Reviewer: ```suggestion
The `HostedControlPlane` resource orchestrates the rollout of the new version of the control plane components along with the {product-title} component in the data plane through the new version of the Cluster Version Operator (CVO). The `HostedControlPlane` includes the following artifacts:
```

* CVO
* Cluster Network Operator (CNO)
* Cluster Ingress Operator

> * * Manifests for the Kube API server, scheduler, and manager

Reviewer: ```suggestion
* Manifests for the Kube API server, scheduler, and manager
```

> * Manifests for the kube API-server (KAS), scheduler, and manager

Reviewer: ```suggestion
* Manifests for the Kube API server, scheduler, and manager
```

* Machine approver
* Autoscaler

> * Infrastructure resources to enable ingress for control plane endpoints such as, KAS, ignition, konnectivity.

Reviewer: ```suggestion
* Infrastructure resources to enable ingress for control plane endpoints such as the Kube API server, ignition, and konnectivity
```
